template/analytics.notify.cve/body.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Security alert notification in {{.Space.Data.Codebase.Name}}</title>
+  <style>
+    tr td ul li img {
+      max-width: 500px;
+    }
+
+    tr td pre {
+      overflow: auto;
+    }
+
+    a:hover {
+      text-decoration: underline !important;
+    }
+
+    table {
+    width:100%;
+    }
+
+    table, th, td {
+        border-collapse: collapse;
+    }
+
+    th, td {
+        padding: 15px;
+        border-bottom: 1px solid #ededed;
+        text-align: left;
+        color: #363636
+    }
+
+    th {
+      font-weight: 500;
+    }
+
+    td {
+      color: #72767B;
+      padding-top: 20px;
+      padding-bottom: 30px;
+    }
+  </style>
+</head>
+
+<body style="
+        margin: 10px;
+        padding: 0; 
+        background-color: #f9f9f9;
+        font-family: 'Open Sans', sans-serif;
+        font-size: 14px;
+        font-weight: lighter;
+        line-height: 1.2;">
+  <div style="
+        min-height: 300px;
+        width: 100%;
+        max-width: 800px;
+        margin: 0 auto;
+        padding: 20px;
+        border: 1px solid #d7d7d7;
+        border-radius: 4px;
+        background-color: #fff;
+        box-shadow: 0 2px 4px #d7d7d7;">
+    <div style="
+          width: 100%;">
+      <img src="{{.webURL}}/_openshiftio/assets/images/openshift_logo_black.png" width="25%" height="45px" />
+      <div style="
+            width: 100%;
+            margin: 10px auto;
+            border: 1px solid #d7d7d7;">
+      </div>
+    </div>
+    <div>
+      <div style="margin-top: 15px;color:#72767B;">
+        <span>
+          <span style="font-weight: 500;color:#363636;">Scan report</span> for Github repository
+          <a href="https://github.com/fabric8-ui/fabric8-ui" style="color: #0088ce; text-decoration: none;">https://github.com/fabric8-ui/fabric8-ui</a>&nbsp;
```

Comment:
Are you sure that you want to use non-breaking spaces here? If the email client window is smaller, this will not wrap like normal text. It might be better to just use a padding?